backend/functions/src/constants/collections.ts

```diff
@@ -0,0 +1,3 @@
+export enum COLLECTION_NAMES {
```

Comment:
Enums are named in singular form, using PascalCase convention. This should be like this:
`CollectionName` or maybe `FirestoreCollection`.

The values can be named in either ALL_CAPS or PascalCase.